docs/pages/cloud-client-ip-restrictions.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: Teleport Cloud Client IP Restrictions
+description: Block CIDR ranges from connecting to your cloud cluster.
+tags:
+ - faq
+ - platform-wide
+---
+
+**Client IP Allowlist** restricts access to your Teleport Cloud cluster, allowing traffic only from the specified network ranges (CIDR blocks).
+
+## How to enable
+This feature is currently opt-in only. Please contact your account executive or [customer support](https://goteleport.com/support/) to enable client IP restrictions for your tenant.
+
+## Adding CIDR blocks to the IP Allowlist
+
+Log in to your Teleport Cloud account. Open the user dropdown menu on the top right of the navigation bar, and select "Help & Support," then scroll down until you see the IP Allowlist section.
+Once you add a CIDR, it will take effect in 5-20 minutes and will terminate existing connections. Changes to the allowlist are recorded in the audit log.
+
+![Web UI view showing IP Allowlist for Teleport](../img/cloud/client-ip-restrictions.png)
+Managing CIDR blocks is governed by Teleport’s existing RBAC system.
+A new resource type is introduced: `client_ip_restriction`.
+To see the current allowlist, the user must have the  `list` verb enabled.
+
+To update the allowlist, `create` and `update` are required.
+
+By default, the `editor` role can **read and write** CIDR allowlists.
+
+## Limitations
+
+### Misconfiguration
+Misconfiguration can block all access to your cluster. Make sure to include your current network before saving changes. 
```

Comment:
I can almost guarantee someone will lock themselves out.

Do we have anything on the roadmap to address this footgun? Most systems with a feature like this will prevent you from blocking your own IP (or at least warn you first).